src/sentry/incidents/logic.py

```diff
@@ -950,14 +950,16 @@ def create_alert_rule_activation_condition(
 
 def create_alert_rule_activation(
     alert_rule: AlertRule,
+    query_subscription_id: int,
```

Comment:
Are there any legacy cases where this might need to be `None`?

Reply:
Nope!
there's no rows currently 🙏